sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -172,41 +176,53 @@ You'll need to create an asynchronous [`ServiceBusReceiverAsyncClient`][ServiceB
 [`ServiceBusReceiverClient`][ServiceBusReceiverClient] to receive messages. Each receiver can consume messages from
 either a queue or a topic subscription.
 
+By default, the receive mode is [`ReceiveMode.PEEK_LOCK`][ReceiveMode]. This tells the broker that the receiving client
+wants to settle received messages explicitly. The message is made available for the receiver to process, while held
```

Comment:
This tells the broker that the receiving client wants to settle received messages explicitly. ->This tells the broker that the receiving client wants to manage settlement of received messages explicitly.